frontend/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styles from './notification.module.css';
+import { notificationBadge, notificationBell } from '../../icons/other';
+
+const Card = () => (
```

Comment:
whys this called card?

Reply:
no idea. I think my editor may have auto-fixed it. will change it to `Notification`